.github/ISSUE_TEMPLATE/release-check-list.md

```diff
@@ -65,73 +53,66 @@ assignees: ''
       * peer-pods: https://github.com/confidential-containers/operator/blob/main/config/samples/ccruntime/peer-pods/kustomization.yaml
           Note that we need the quay.io/confidential-containers/runtime-payload-ci registry and kata-containers-latest tag
 
-8. - [ ] Update peer-pods with latest commits of kata-containers and attestation-agent and test it, following the [release candidate testing process](https://github.com/confidential-containers/cloud-api-adaptor/blob/main/docs/Release-Process.md#release-candidate-testing)
+6. - [ ] Update peer-pods with latest commits of kata-containers and attestation-agent and test it, following the [release candidate testing process](https://github.com/confidential-containers/cloud-api-adaptor/blob/main/docs/Release-Process.md#release-candidate-testing)
     
-9. - [ ] Cut an ocicrypt-rs v<TARGET_RELEASE> release, if changes happened in the project
-
-10. - [ ] Cut an attestation-agent v<TARGET_RELEASE>, if changes happened in the project
-
-11. - [ ] Cut an attestation-service v<TARGET_RELEASE> and make images for AS and RVPS, if changes happened in the project.
+7. - [ ] Cut an attestation-service v<TARGET_RELEASE> and make images for AS and RVPS, if changes happened in the project.
 
        * https://github.com/confidential-containers/attestation-service
        * Cut a release (AS/RVPS images will be automatically built triggered by release)
 
-12. - [ ] Update kbs to use the latest commit from attestation-service, cut a release and make image
-
-       * https://github.com/confidential-containers/kbs/blob/main/src/api_server/Cargo.toml
-       * Change the revision for the following crates (both use `v<TARGET_RELEASE>`)
-     * `as-types`
-     * `attestation-service`
-       * Cut a release (kbs image will be automatically built triggered by release)
+8. - [ ] Cut a guest-components v<TARGET_RELEASE> release
 
-13. - [ ] Cut an image-rs v<TARGET_RELEASE> release, using the latest release of:
+9. - [ ] Cut a td-shim v<TARGET_RELEASE> release, if changes happened in the project
 
-    * ocicrypt-rs (redo step 1, but now using v<TARGET_RELEASE>)
-    * attestation-agent (redo step 2, but now using v<TARGET_RELEASE>)
+10. - [ ] Update kbs to use the tagged attestation-service and guest-components, cut a release and make image
 
-14. - [ ] Cut a td-shim v<TARGET_RELEASE> release, if changes happened in the project
+       * https://github.com/confidential-containers/kbs/blob/main/src/api/Cargo.toml
+         * Change the revision for the `as-types` and `attestation-service` crates (both use `v<TARGET_RELEASE>`)
+       * https://github.com/confidential-containers/kbs/blob/main/tools/client/Cargo.toml
+         * Change the revision for the `kbs_protocol` crate (use `v<TARGET_RELEASE>`)
```

Comment:
Also need to update `as-types` for the the client.

Reply:
done